server/src/controllers/addJob.ts

```diff
@@ -0,0 +1,32 @@
+import { AuthRequest } from '../interfaces';
+import { Job } from '../models';
+import { addJobValidation } from '../validation';
+
+const addJob = async (req: AuthRequest) => {
+  const {
+    title,
+    budget,
+    time,
+    category,
+    description,
+  } = req.body;
+  const userId = req.user?.userID;
```

Comment:
```suggestion
  const userId = req.user?.userID;
```
this is not `req.user` anymore